Buildings/Controls/OBC/ASHRAE/G36/AHUs/MultiZone/SetPoints/ReliefDamper.mo

```diff
@@ -0,0 +1,143 @@
+    annotation (Line(points={{-39,40},{-22,40}}, color={0,0,127}));
+  connect(uBuiPre, damPosController.u_m)
+    annotation (Line(points={{-100,20},{-10,20},{-10,28}},
+      color={0,0,127}));
+  connect(uSupFan, swi.u2)
+    annotation (Line(points={{-100,-20},{-32,-20},{-32,0},{38,0}},
+      color={255,0,255}));
+  connect(damPosController.y, swi.u1)
+    annotation (Line(points={{1,40},{20,40},{20,8},{38,8}},
+      color={0,0,127}));
+  connect(zerDam.y, swi.u3)
+    annotation (Line(points={{-39,-40},{20,-40},{20,-8},{38,-8}},
+      color={0,0,127}));
+  connect(swi.y, yRelDamPos)
+    annotation (Line(points={{61,0},{90,0}},  color={0,0,127}));
+
+annotation (
+  defaultComponentName = "reliefDamper_multiZone",
+  Icon(graphics={
+        Rectangle(
+        extent={{-100,-100},{100,100}},
+        lineColor={0,0,127},
+        fillColor={255,255,255},
+        fillPattern=FillPattern.Solid),
+        Text(
+          extent={{-96,78},{-56,42}},
+          lineColor={0,0,127},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          textString="uBuiPre"),
+        Text(
+          extent={{-96,-42},{-52,-78}},
+          lineColor={0,0,127},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          textString="uSupFan"),
+        Text(
+          extent={{34,22},{96,-18}},
+          lineColor={0,0,127},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          textString="yRelDamPos"),
+        Polygon(
+          points={{-80,92},{-88,70},{-72,70},{-80,92}},
+          lineColor={192,192,192},
+          fillColor={192,192,192},
+          fillPattern=FillPattern.Solid),
+        Line(points={{-80,80},{-80,-88}}, color={192,192,192}),
+        Line(points={{-90,-78},{82,-78}}, color={192,192,192}),
+        Polygon(
+          points={{90,-78},{68,-70},{68,-86},{90,-78}},
+          lineColor={192,192,192},
+          fillColor={192,192,192},
+          fillPattern=FillPattern.Solid),
+        Line(points={{-80,-78},{-80,-78},{14,62},{80,62}}, color={0,0,127}),
+        Text(
+          extent={{-100,140},{100,100}},
+          lineColor={0,0,255},
+          textString="%name")}),
+    Diagram(
+        coordinateSystem(preserveAspectRatio=false, extent={{-80,-80},{80,80}})),
+ Documentation(info="<html>      
+<p>
+This sequence controls actuated relief dampers (<code>yRelDamPos</code>) 
+without fans. It is implemented according to ASHRAE Guidline 35 (G36), PART5.N.8. 
+(for multiple zone VAV AHU), PART5.P.6 and PART3.2B.3 (for single zone VAV AHU).
+</p>   
+<h4>Multiple zone VAV AHU: Control of actuated relief dampers without fans(PART5.N.8)</h4>
+<ol>
+<li>Relief dampers shall be enabled when the associated supply fan is proven on 
+(<code>uSupFan = true</code>), and disabled otherwise.</li>
+<li>When enabled, use a P-only control loop to modulate relief dampers to maintain 
+0.05” (12 Pa) building static pressure (<code>uBuiPre</code>). 
```

Comment:
better write "inches" rather than quotes, as quotes can cause issues between different tools that process text. or use ``&quot;``